src/style.css

```
@@ -1,3 +1,75 @@
-body {
+:root {
+  --blue-color: #00acdc;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  background-color: #fff;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+}
+
+.list {
+  list-style: none;
+  display: flex;
+  align-items: center;
+}
+
+._link {
+  display: flex;
+  height: 60px;
+  align-items: center;
+  text-decoration: none;
+  margin: 0 10px;
+  color: black;
+  text-transform: uppercase;
```

you can split your styles for better code style for exm.: 
```suggestion
  display: flex;
  align-items: center;
  
  margin: 0 10px;
  height: 60px;
  
  color: black;
  text-decoration: none;
  text-transform: uppercase;
```